src/vivarium_public_health/risks/base_risk.py

```diff
@@ -252,11 +252,7 @@ def get_randomness_stream(self, builder: Builder) -> RandomnessStream:
     def get_propensity_pipeline(self, builder: Builder) -> Pipeline:
```

Comment:
This is a classic example of where we previously wanted the functionality we now have. We can get rid of this explicit pipeline registration by renaming the propensity column name (`f"{self.risk.name}_propensity"`) to the propensity pipeline name (`f"{self.risk.name}.propensity"`).